### PR TITLE
Add better naming to export json

### PR DESCRIPTION
**Pull request by siddhant1**

### CODE REVIEW

File: src/components/header/export-modal/renderer.tsx [hunk 1]
```diff
@@ -159,43 +167,49 @@ class ExportModal extends React.PureComponent<Props, State> {
...
-                  type="checkbox"
-                  name="config-include"
-                  id="config-include"
-                  value="config-select"
-                  checked={this.state.includeConfig}
-                  onChange={this.updateIncludeConfig.bind(this)}
-                />
-              </label>
+              {this.state.downloadVegaJSON ? (
+                <div>The compiled Vega includes the config</div>
```
Comment (domoritz) [inline src/components/header/export-modal/renderer.tsx]: you forgot the . we talked about

**Comment by domoritz** [comment]
Let me know when this is ready.

**Comment by siddhant1** [comment]
Its ready from my side, please check and verify for mistakes!

**Comment by domoritz** [comment]
You forgot to add the note we talked about. When the config is being included, then the spec will be re-formatted.

**Pull request merged**